zerver/models.py

```diff
@@ -1147,7 +1147,93 @@ def __str__(self) -> str:
         return f"<Recipient: {display_recipient} ({self.type_id}, {self.type})>"
 
 
-class UserProfile(AbstractBaseUser, PermissionsMixin):
+class UserBaseSettings(models.Model):
+    # display settings
```

Comment:
We should add a block comment here explaining the purpose of this model a bit, and also mentioning that whenever we change a value declared here, we may need to do a migration to change that value for all existing RealmUserDefault objects as well, if we want that change in defaults to affect new users in existing organizations?